lib/busbar_cli/services/logs.rb

```diff
@@ -1,14 +1,72 @@
 module Services
   class Logs
-    def self.call(container_id:, environment_name:, since:)
+    def self.call(app_id:, environment_name:, component_id:, since:, size:)
       Services::Kube.setup
+      require 'open3'
 
-      container_name = container_id.split('-')[0..-3].join('-')
+      @environment_name = environment_name
+      @app_id = app_id
+      @component_id = component_id
 
-      Kernel.exec(
-        "#{KUBECTL} --context=#{Services::Kube.current_profile} " \
-        "logs -f --since=#{since} #{container_id} #{container_name} -n #{environment_name}"
+      # Retrieve Component Data
+      # TODO: Refactor this into a service => https://github.com/busbar-io/busbar-cli/issues/45
```

Comment:
Personally, I'm not a big fan of `TODO`s in production-ready code (i.e., in `master`). Since you've already created [an issue for this](https://github.com/busbar-io/busbar-cli/issues/45), what do you think of referencing this code block on the issue itself, so that you can quickly identify where to make the changes when the time comes for it?

Reply:
Oh! Neat idea! doing it! =))